Azure-Samples/azure-search-openai-demo
# Add token validation

## Purpose

* Validate authentication tokens before calling Microsoft.Graph on behalf of the user

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Enable authentication
* Validate all endpoints that require authentication still work with your client app registration
* Re-run auth_init.ps1/sh and confirm the authentication still works even after a token version upgrade

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```
